tests/foreman/cli/test_docker.py

```diff
@@ -1214,218 +1216,352 @@ class DockerContainersTestCase(CLITestCase):
     def setUpClass(cls):
         """Create an organization and product which can be re-used in tests."""
         super(DockerContainersTestCase, cls).setUpClass()
-        cls.org_id = entities.Organization().create_json()['id']
-        # TODO: create Docker-based compute resources (internal/external)
+        cls.org = make_org()
+        cls.cr_internal = make_compute_resource({
+            'organization-ids': [cls.org['id']],
+            'provider': DOCKER_PROVIDER,
+            'url': INTERNAL_DOCKER_URL,
+        })
+        cls.cr_external = make_compute_resource({
+            'organization-ids': [cls.org['id']],
+            'provider': DOCKER_PROVIDER,
+            'url': EXTERNAL_DOCKER_URL,
+        })
 
-    @stubbed()
-    @run_only_on('sat')
-    def test_create_container_local_compute_resource(self):
-        """@Test: Create a container in a local compute resource
+    def test_create_container(self):
+        """@Test: Create containers for local and external compute resources
 
         @Feature: Docker
 
-        @Assert: The docker container is created in the local compute resource
-
-        @Status: Manual
+        @Assert: The docker container is created for each compute resource
 
         """
+        for compute_resource in (self.cr_internal, self.cr_external):
+            with self.subTest(compute_resource['url']):
+                container = make_container({
+                    'compute-resource-id': compute_resource['id'],
+                    'organization-ids': [self.org['id']],
+                })
+                self.assertEqual(
+                    container['compute-resource'], compute_resource['name'])
 
-    @stubbed()
-    @run_only_on('sat')
-    def test_create_container_external_compute_resource(self):
-        """@Test: Create a container in an external compute resource
+    @skip_if_bug_open('bugzilla', 1230915)
+    @skip_if_bug_open('bugzilla', 1269196)
+    def test_container_power(self):
+        """@Test: Create containers for local and external compute resource,
+        then power them on and finally power them off
 
         @Feature: Docker
 
-        @Assert: The docker container is created in the external compute
-        resource
+        @Assert: The docker container is created for each compute resource
+        and the power status is showing properly
 
-        @Status: Manual
+        @BZ: 1230915, 1269196
 
         """
-
-    @stubbed()
-    @run_only_on('sat')
-    def test_create_container_local_compute_resource_power(self):
-        """@Test: Create a container in a local compute resource, then power it
-        on and finally power it off
+        not_running_msg = 'Running: no'
```

Comment:
I know that fact and I was hoping to avoid. Unfortunately, status command just returns this string and that's it.
Even more, command is broken for now and always returns 'Running: no' for any status (BZ1269196), so looks like no other options for now :/